src/pages/articles/components/create/articleCreate.tsx

```diff
@@ -64,16 +63,17 @@ export default function ArticleCreate() {
     setter(value === '' ? undefined : parseFloat(value));
   };
 
-  // 포스트 생성
-  const createPost = async () => {
+  // 아티클 생성
+  const createArticle = async () => {
     if (!validateRequiredFields()) {
       return;
     }
-    const html = editorRef.current?.getInstance().getHTML() || '';
+    const markdownContent =
+      editorRef.current?.getInstance().getMarkdown() || '';
```

Comment:
```suggestion
    const markdownContent =
      editorRef.current?.getInstance().getMarkdown() || '';
```

@sw61 옵셔널 연산자 대신 ref객체 존재 여부를 코드 상단에서 확인 후 early return을 하는 방식은 어떨까요?

Reply:
그 방식은 생각을 못했네요. 성능 측면에서도 그 방식이 훨씬 좋을 것 같아요.